### PR TITLE
Fix media double-click handler

### LINKED ISSUE #6163: Unable to enter Media edit mode
## Bug Description
Entering media edit mode via double click is not working.

## Expected Behaviour
Double-click on media element should enter edit mode.

## Steps to Reproduce
1. Add media
2. Double-click on it
3. See nothing happening

## Screenshots
edit

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by miina**
## Context
The media double-click handler was never used due to the click event not reaching there.
This PR moves the click handler together with the general `mouseDown` handler.

## Testing Instructions
1. Add media (try with video and image, background and regular element)
2. Try entering edit mode via double-click, verify it works.

### QA

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1. Add media (try with video and image, background and regular element)
2. Try entering edit mode via double-click, verify it works.

### UAT

- [x] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testiing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #6163

### CODE REVIEW

File: assets/src/edit-story/components/canvas/frameElement.js [hunk 1]
```diff
@@ -108,6 +112,23 @@ function FrameElement({ element }) {
     setIsTransforming(transform !== null);
   });
 
+  // Media needs separate handler for double click.
+  const { isMedia } = getDefinitionForType(type);
+  const handleSingleClick = useCallback(() => {}, []);
```
Comment (barklund) [inline assets/src/edit-story/components/canvas/frameElement.js]: Hm, I'm sure we have a `NOOP` somewhere. Or you could just define it as a constant in this file as `const NOOP = () => {};` and use that directly in the `useDoubleClick` hook below.

Comment (barklund) [reply assets/src/edit-story/components/canvas/frameElement.js]: (I do realize you're just moving code, but still)

**Review by barklund — APPROVED** [review]
Looks good. This doesn't impact text edit, right?

**Review by swissspidy — APPROVED** [review]

**Comment by miina** [comment]
> This doesn't impact text edit, right?

Correct.

**Comment by swissspidy** [comment]
#6162 disabled this karma test, so need to re-enable it again here.

**Pull request merged**